src/Microsoft.VisualStudio.ProjectSystem.FSharp.VS/Microsoft.VisualStudio.ProjectSystem.FSharp.VS.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
 <Project Sdk="Microsoft.NET.Sdk">
-  <Import Project="..\VisualStudio.props"/>
+  <Import Project="..\VisualStudio.props" />
```

Comment:
Nit: no meaningful changes in this file - revert the changes?

Reply:
Yea. Looks like it auto-formatted it.